orb-attest/src/main.rs

```diff
@@ -1,4 +1,5 @@
 #[tokio::main]
-async fn main() -> eyre::Result<()> {
+async fn main() -> color_eyre::Result<()> {
+    color_eyre::install()?;
```

Comment:
wouldn't that break logging via journalctl?

Reply:
Can you elaborate how would it break?

Reply:
My concern is mostly called by my ignorance of `color_eyre` inner workings. 

In the  [orb_attest::main](https://github.com/worldcoin/orb-software/blob/3d369d9cace3b4fdf3f4f3fa97629fb431c4774e/orb-attest/src/lib.rs#L25) we [initialize logger ](https://github.com/worldcoin/orb-software/blob/3d369d9cace3b4fdf3f4f3fa97629fb431c4774e/orb-attest/src/logging.rs#L46) at that moment it decides where to send logs, to stdout or to journalctl. 

I was worried that `color_eyre` might interfere with the configuration of the logger. For example it might print something when the service is expected to use journalctl.

Reply:
Here are some logs I just captured from journalctl:
```
Jul 19 23:57:52 localhost systemd[1]: worldcoin-attest.service: Main process exited, code=exited, status=1/FAILURE
Jul 19 23:57:52 localhost systemd[1]: worldcoin-attest.service: Failed with result 'exit-code'.
Jul 19 23:57:52 localhost systemd[1]: worldcoin-attest.service: Scheduled restart job, restart counter is at 4.
Jul 19 23:57:52 localhost systemd[1]: Stopped Short lived attestation token daemon.
Jul 19 23:57:52 localhost systemd[1]: Started Short lived attestation token daemon.
Jul 19 23:57:52 localhost orb-attest[3353]: Version: 0.2.9
Jul 19 23:57:52 localhost orb-attest[3353]: git sha: 8a66e79
Jul 19 23:57:52 localhost orb-attest[3353]: Error:
Jul 19 23:57:52 localhost orb-attest[3353]:    0: Initialization failed
Jul 19 23:57:52 localhost orb-attest[3353]:    1: foobar
Jul 19 23:57:52 localhost orb-attest[3353]: Location:
Jul 19 23:57:52 localhost orb-attest[3353]:    /Users/ryan.butler/P/wc/orb-software/orb-attest/src/lib.rs:113
Jul 19 23:57:52 localhost orb-attest[3353]: Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Jul 19 23:57:52 localhost orb-attest[3353]: Run with RUST_BACKTRACE=full to include source snippets.
Jul 19 23:57:52 localhost orb-attest[3353]: Warning: SpanTrace capture is Unsupported.
Jul 19 23:57:52 localhost orb-attest[3353]: Ensure that you've setup a tracing-error ErrorLayer and the semver versions are compatible
```